content/doc/find-help/troubleshooting.md

```diff
@@ -169,3 +169,16 @@ You can of course reach to <support@clever-cloud.com> if this page was not helpf
 
 Check the status of the services on [clevercloudstatus.com](https://www.clevercloudstatus.com/). The support team is also there to help you identify any potential errors or misconfiguration.
 {{% /details %}}
+
+{{% details title="There are a lot of open ports, according to a port scan." closed="true" %}}
+
+If you tried to scan your domain, you may not only have scanned your app, but also several public load balancers.
+Here is an explanation of the different port you may see.
+
+- Port 9999 : Zabbix use this port to monitor the hypervisor.
```

Comment:
Fix grammar and add link to Zabbix homepage, so readers know what we are talking about.

```suggestion
- Port 9999 : [Zabbix](https://www.zabbix.com) uses this port to monitor the hypervisor.
```